src/api/api.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 
 import { getPartnerAttributionID, getSessionID } from "@paypal/sdk-client/src";
-import { request } from "@krakenjs/belter/src";
+import { inlineMemoize, request } from "@krakenjs/belter/src";
```

Comment:
what is the difference between [`inlineMemoize`](https://github.com/krakenjs/belter/blob/d0c0868c996aea92d1ad9a6c8662da7a2c200b5b/src/util.js#L334) and [`memoize`](https://github.com/krakenjs/belter/blob/d0c0868c996aea92d1ad9a6c8662da7a2c200b5b/src/util.js#L188)

just curious..

Reply:
oh reading below it seems like it will cache based on the args passed

Reply:
Great question. You are 💯 right about args. Although memoize also creates cacheKey based on method arguments, but inlineMemoize generally provides more control on the cache key e.g. we can serialize complex objects as per the needs. 

Also with inlineMemoize the cache lives with method but memoize will end up creating a closure for the cache.